### PR TITLE
Added features

### PR DESCRIPTION
**Pull request by thankgodr**
## This PR contains

- methods related to the status updates
- Clear all completed" button (use filter() method
- All project requirements
- Code lint

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -13,7 +14,7 @@
         <div class="p-2">
             <div class="listContent">
                 <div class="row">
-                    <div class="col-md-12"">
+                    <div class="col-md-12">
                         <div class="col-md-8">
                             <h3>Today's To DO</h3>
```
Comment (Kamba56) [inline src/index.html:19]: - [Optional] Consider using an indentation of two spaces in all your documents. This is a convention and also [best practice.](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)

File: src/js/index.js [hunk 2]
```diff
@@ -36,18 +36,23 @@ const printAllTasks = () => {
       taskmanager.editTask(task);
     });
```
Comment (Kamba56) [inline src/js/index.js]: - [ ] Please as demanded by the project requirements, the functionalities for the status updates (completed: true / false) should be in a separate js file. Consider moving this to a new method in a new js file and import then call the method here.
PS: **I couldn't comment on the particular lines that's why I commented below but I am talking about line 31-37**

**Review by Kamba56 — CHANGES_REQUESTED** [review]
Hi @thankgodr, 

Good job so far!!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- [x] Great use of Github workflow!👏👏👏
- [x] All linters passed

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ichala — APPROVED** [review]
Hi @thankgodr ,
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:  
 
68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f6c3366516333614356304871707a5763552f67697068792e676966

# To Highlight 💪🏻

-Requirements Implemented ✔️ 


 Good job implementing the requirements  :+1: Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Optional Suggestions  :

- N/A


**********
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag  @ichala in your question so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**